src/chia_dialect.rs

```diff
@@ -21,6 +21,10 @@ pub const LIMIT_HEAP: u32 = 0x0004;
 // When set, enforce a stack size limit for CLVM programs
 pub const LIMIT_STACK: u32 = 0x0008;
 
+// When set, we allow softfork with extension 0 (which includes coinid and the
+// BLS operators)
+pub const ENABLE_BLS_OPS: u32 = 0x0010;
```

Comment:
Ah, I see, as we move through the blockchain consensus, we have to adjust how the softfork operator works.

I feel like this can be completely encapsulated in `Dialect`. For simplicity, suppose a soft fork kicks in that introduces a new dialect that supports `coinid`.